replset-consistency/README.md

```diff
@@ -232,7 +244,7 @@ When remediation is complete, resume writes to the collection(s) being remediate
 
 ## 5. Resolve any remaining index inconsistencies
 
-Now that document data is confirmed consistent, and if `validate{}` previously indicated index inconsistencies, perform an initial sync of all affected nodes in sequence, to ensure indexes are rebuilt.
+Now that document data is confirmed consistent, and if `validate()` previously indicated index inconsistencies, perform an [initial sync](https://www.mongodb.com/docs/manual/core/replica-set-sync/) of all affected nodes in sequence, to ensure indexes are rebuilt.
```

Comment:
should we explicitly mention that initial sync will resolve index consistencies?

Reply:
Is this asking to remove "previously indicated index inconsistencies" and "to ensure indexes are rebuilt?"
If so, I'm OK with that, just unsure.

Reply:
No, I think we should clarify the instructions remediating between local and replica set inconsistencies. 

I saw some verbiage earlier in the README you may consider reusing for remediating local inconsistencies:
> any nodes that fail validation should be initial-synced from a node that passes validation

After remediating replica set inconsistencies, we can use any node as an initial sync source, as long as the original sync source is also initially synced at the very end.